docs/source/installation.md

```diff
@@ -1,13 +1,30 @@
 (installation)=
 # Installation
 
-To install PyMC, select the operating system where you want to perform the installation.
+We recommend using [Anaconda](https://www.anaconda.com/) (or [Miniforge](https://github.com/conda-forge/miniforge)) to install Python on your local machine, which allows for packages to be installed using its `conda` utility.
 
-## Linux
-[Linux installation guide](https://github.com/pymc-devs/pymc/wiki/Installation-Guide-(Linux))
+Once you have installed one of the above, PyMC can be installed into a new conda environment as follows:
 
-## MacOS
-[MacOS installation guide](https://github.com/pymc-devs/pymc/wiki/Installation-Guide-(MacOS))
+```console
+conda create -c conda-forge -n pymc_env pymc
+conda activate pymc_env
+```
+If you like, replace the name `pymc_env` with whatever environment name you prefer.
+
+
+## JAX sampling
+
+If you wish to enable sampling using the JAX backend via NumPyro (experimental), the following should also be installed:
```

Comment:
is only numpyro experimental but not blackjax? If so I'd change the order and put blackjax first

Reply:
I assumed it was experimental until it was integrated into `pm.sample` ... happy to drop it though,